components/Conversation.tsx

```diff
@@ -11,13 +11,13 @@ const ConversationContainer = styled.div`
 `;
 
 const Conversation = ({ localParticipant, participants }) => {
+  const { query } = useRouter();
```

Comment:
Explanation from my commit:

------------

According to https://reactjs.org/docs/hooks-rules.html:

>Rule #1: Don’t call Hooks inside loops, conditions, or nested functions.

This was causing a React warning in the console because the hook order
changed between renders. Now it won't.